st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -156,13 +156,16 @@ def post(self, action_alias):
 
         return action_alias_api
 
-    @request_user_has_resource_db_permission(permission_type=PermissionType.ACTION_MODIFY)
+    @request_user_has_resource_db_permission(permission_type=PermissionType.ACTION_ALIAS_MODIFY)
     @jsexpose(arg_types=[str], body_cls=ActionAliasAPI)
-    def put(self, action_alias_ref_or_id, action_alias):
+    def put(self, action_alias, action_alias_ref_or_id):
```

Comment:
In st2api v2, position of the argument would not matter at all, all of them will be passed as kwargs.